### PR TITLE
Added equality/comparison overloads for Duration and TimeSpan

### PR DESCRIPTION
**Pull request by eriove**
Added equality/comparison overloads for Duration and TimeSpan. This makes it easier to work seamlessly with both types in the same code.

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/DurationTests.cs [hunk 1]
```diff
@@ -111,5 +111,101 @@ public static void DateTimeMinusDurationReturnsDateTime()
...
+            Duration duration = Duration.FromHours(11);
+            Assert.IsTrue(duration != timeSpan, "duraiton should equal to timeSpan");
+        }
+
+        [Test]
+        public static void TimeSpanNotEqualToDurationShouldReturnCorrectly()
+        {
+            TimeSpan timeSpan = TimeSpan.FromHours(12);
+            Duration duration = Duration.FromHours(11);
+            Assert.IsTrue(timeSpan != duration, "timeSpan should equal to duraiton");
```
Comment (anjdreas) [inline UnitsNet.Tests/CustomCode/DurationTests.cs]: A couple of typos on `duraiton` here.

**Comment by anjdreas** [comment]
Other than that, looks good. Let me know when fixed and I will merg it in.

**Comment by eriove** [comment]
Should be fixed now.

**Comment by anjdreas** [comment]
Thanks. Also seems github finally introduced email notifications on new commits in a PR :-)

**Comment by anjdreas** [comment]
Nuget out.

**Pull request merged**